packages/module/patternfly-docs/content/extensions/component-groups/examples/ContentHeader/ContentHeaderLabelLinkExample.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ContentHeader from '@patternfly/react-component-groups/dist/dynamic/ContentHeader';
+
+export const BasicExample: React.FunctionComponent = () => (
+  <ContentHeader 
+    title='My Title'
+    subtitle='This is a subtitle for your content header'
+    label='Org. Administrator'
```

Comment:
This probably needs to be updated to display the PF Label in the example, not a plain string